### PR TITLE
[OSCD] adding shortened commands for Netsh in the existing rule

### PR DESCRIPTION
**Pull request by omkar72**

### CODE REVIEW

**Comment by e6e6e** [comment]
@omkar72 
Please change the line **_modified_**.

**Comment by omkar72** [comment]
@e6e6e Thanks for catching that

**Comment by e6e6e** [comment]
You have made the correct changes. =)
I think it's ready to be merged. @yugoslavskiy

File: rules/windows/process_creation/win_netsh_port_fwd.yml [hunk 1]
```diff
@@ -4,22 +4,29 @@ description: Detects netsh commands that configure a port forwarding
...
-author: Florian Roth
+author: Florian Roth, omkar72, oscd.community
 logsource:
     category: process_creation
     product: windows
 detection:
-    selection:
+    selection1:
         CommandLine:
             - netsh interface portproxy add v4tov4 *
```
Comment (yugoslavskiy) [inline rules/windows/process_creation/win_netsh_port_fwd.yml:21]: could you please change this line to the following:

```
Image|endswith: 
            - '\netsh.exe'
        CommandLine|contains|all:
            - 'interface'
            - 'portproxy'
            - 'add'
            - 'v4tov4'
```

**Comment by omkar72** [comment]
changed conditions as suggested

**Comment by yugoslavskiy** [comment]
Thank you very much, @omkar72! Well done!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**